script.py

```diff
@@ -100,24 +118,36 @@ def cleanup_backups(bucket):
     four_days_ago = datetime.now() - timedelta(days=4)
```

Comment:
pick the days=4 from env vars
make an env var called `go_back_n_days` or `n_days_ago` whatever you think best